src/Sylius/Bundle/AdminBundle/templates/customer/show/component/order_statistics.html.twig

```diff
@@ -1,6 +1,6 @@
-{% from '@SyliusAdmin/shared/helper/accordion.html.twig' import default as _accordion %}
-{% from '@SyliusAdmin/shared/helper/icon.html.twig' import icon as _icon %}
-{% from '@SyliusAdmin/shared/helper/money.html.twig' import format as _money %}
+{% from '@SyliusAdmin/shared/helper/accordion.html.twig' import default as accordion %}
+{% from '@SyliusAdmin/shared/helper/icon.html.twig' import icon as icon %}
+{% from '@SyliusAdmin/shared/helper/money.html.twig' import format as money %}
```

Comment:
We have an inconsistency with the money macro, sometimes it's importing `format` directly other times it's accessing through money: `money.format(...)`